recipes/jemalloc/all/conanfile.py

```diff
@@ -52,8 +52,10 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.compiler.get_safe("libcxx") == "libc++":
-            raise ConanInvalidConfiguration("libc++ is missing a mutex implementation.  Remove this when it is added")
+        if self.settings.compiler.get_safe("libcxx") == "libc++" and \
+                self.settings.compiler == "clang" and \
+                tools.Version(self.settings.compiler.version) < "10":
+            raise ConanInvalidConfiguration("old libc++ versions are missing a mutex implementation")
```

Comment:
nit: mention the libc++ version within the message. Otherwise you have to read the recipe code to know what "old" means